lib/jormungandr/src/Cardano/Wallet/Jormungandr.hs

```diff
@@ -447,3 +427,69 @@ exitCodeApiServer = \case
     ListenErrorInvalidAddress _ -> 11
     ListenErrorAddressAlreadyInUse _ -> 12
     ListenErrorOperationNotPermitted -> 13
+
+{-------------------------------------------------------------------------------
+                                    Tracers
+-------------------------------------------------------------------------------}
+
+-- | The type of trace events produced by the Jörmungandr API server.
+data JormungandrServerLog
+    = MsgLogServer ApplicationLog
+    | MsgApiServer ApiLog
+    | MsgApiWorker WorkerRegistryLog
+    | MsgWalletDB WalletDBLog
+    | MsgDBLog DBLog
+    | MsgNetwork NetworkLayerLog
+    | MsgStakePoolMonitor StakePoolMonitorLog
+    | MsgStakePoolLayer StakePoolLayerLog
+    | MsgStakePoolDB DBLog
+    | MsgDaedalusIPC DaedalusIPCLog
+    | MsgWorkerRegistry WorkerRegistryLog
+    deriving (Show)
+
+instance ToText JormungandrServerLog where
+    toText = \case
+        MsgLogServer msg -> toText msg
+        MsgApiServer msg -> toText msg
+        MsgApiWorker msg -> toText msg
+        MsgWalletDB msg -> toText msg
+        MsgDBLog msg -> toText msg
+        MsgNetwork msg -> toText msg
+        MsgStakePoolLayer msg -> toText msg
+        MsgStakePoolMonitor msg -> toText msg
+        MsgStakePoolDB msg -> toText msg
+        MsgDaedalusIPC msg -> toText msg
+        MsgWorkerRegistry msg -> toText msg
+
+
+-- | The type of trace events produced by the Jörmungandr API server.
+data Tracers' f = Tracers
+    { applicationTracer :: f ApplicationLog
+    , apiServerTracer :: f ApiLog
+    , stakePoolLayerTracer :: f StakePoolLayerLog
+    , dbLayerTracer :: f DBLog
+    , daedalusIPCTracer :: f DaedalusIPCLog
+    }
```

Comment:
**[remark]**

I am not sure to get the choice of names here :thinking: ? 
`dbLayer` refers to both the stake pools database and the wallet database (which we might want to distinguish I suppose). Also, the logs from the network seems missing, oversight or on purpose ?

Reply:
I revised the list of tracers.